src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -892,5 +892,22 @@ configurationRegistry.registerConfiguration({
 			enum: ['always', 'never', 'fromEditor'],
 			default: 'fromEditor'
 		},
+		[NotebookSetting.outputLineHeight]: {
+			markdownDescription: nls.localize('notebook.outputLineHeight', "Line height of the output text for notebook cells.\n - Values between 0 and 8 will be used as a multiplier with the font size.\n - Values greater than or equal to 8 will be used as effective values."),
```

Comment:
'Values between 0 and 8' are multipliers? I assume this is already an existing thing? Super weird.

Reply:
same way that Editor.lineHeight works